lib/matplotlib/mathtext.py

```diff
@@ -189,6 +189,7 @@ class MathtextBackendPs(MathtextBackend):
         "_PSResult", "width height depth pswriter used_characters")
 
     def __init__(self):
+        super().__init__()
```

Comment:
Are none of these really ever tested?  Maybe beyond this PR, but they should be if they aren't...

Reply:
They're almost all unused, which I guess is why they're deprecated. For the ones that _are_ used, I assume they all get `.set_canvas_size` called first, which sets the things that were missed in the super `__init__`.